db/people.yml

```diff
@@ -1980,10 +1980,30 @@ jumana:
   position: undergraduate research assistant
 jwilley:
   active: true
-  aka: []
-  avatar: "https://github.com/jwilley44.png"
-  bio: bio
-  education: []
+  aka:
+    - John J. Willey
+    - John Willey
+  avatar: https://en.gravatar.com/userimage/47018931/e47a250619adbfe7d9857ce3d628b837.jpg
+  bio: John Willey is currently pursuing a master's degree in applied physics. He has 
+    been a data scientist and software engineer in biotech since 2008. His research interests
+    include solid state physics, quantum computing, and quantum optics. Outside of physics he 
+    enjoys spending time with his wife, daughter, and dog.
+  education:
+    - institution: Columbia University
+      begin_year: 2019
+      location: New York, NY
+      department: Applied Physics
+      degree: MS in Applied Physics
+    - institution: University of California Santa Cruz
```

Comment:
to minimize errors I dereference institutions from another institutions collection.  What you have here is fine, but since I asked for another edit below, can I ask you to replace with `ucsantacruz` and make sure that `ucsantacruz` is all correct in and present in the institutions collection?  It is a service to the greater good....Thanks so much.